packages/components-js/tests/e2e/specs/carousel.e2e.ts

```diff
@@ -610,6 +634,29 @@ test.describe('viewport change', () => {
     await waitForStencilLifecycle(page);
     expect(await pagination.evaluate((el) => el.children.length)).toBe(4);
   });
+
+  test('should update pagination for BreakpointCustomizable in focusOnCenterSlide mode', async ({ page }) => {
```

Comment:
The problem here wasn't the breakpoint customisable but rather the dynamic change to the `slidesPerPage`. Do would be enough to just change this value dynamically.

Reply:
Plz re-check :-)